ios/RCTConvert+BitmovinPlayer.swift

```diff
@@ -1177,6 +1183,87 @@ extension RCTConvert {
         ]
     }
 
+    static let iso8601Formatter: ISO8601DateFormatter = {
+        let formatter = ISO8601DateFormatter()
+        formatter.formatOptions = [.withInternetDateTime, .withFractionalSeconds]
+        return formatter
+    }()
+
+    static func toJson(avMetadataItem: AVMetadataItem, metadataType: MetadataType) -> [String: Any] { // swiftlint:disable:this cyclomatic_complexity line_length
+        var json: [String: Any] = [
+            "metadataType": metadataTypeString(metadataType),
+            "platform": "ios"
+        ]
+
+        // Time
+        if let startTime = avMetadataItem.time.safeSeconds {
+            json["startTime"] = startTime
+        }
+        if let duration = avMetadataItem.duration.safeSeconds {
+            json["duration"] = duration
+        }
+
+        // Identifier
+        if let keySpace = avMetadataItem.keySpace?.rawValue {
+            json["keySpace"] = keySpace
+        }
+        if let identifier = avMetadataItem.identifier?.rawValue {
+            json["id"] = identifier
+        }
+        if let commonKey = avMetadataItem.commonKey?.rawValue {
+            json["commonKey"] = commonKey
+        }
+
+        // Language
+        if let extendedLanguageTag = avMetadataItem.extendedLanguageTag {
+            json["extendedLanguageTag"] = extendedLanguageTag
+        }
+        if let localeIdentifier = avMetadataItem.locale?.identifier {
+            json["localeIdentifier"] = localeIdentifier
+        }
+
+        // Value
+        if let dataType = avMetadataItem.dataType {
+            json["dataType"] = dataType
+        }
+        if let valueJson = toValueJson(avMetadataItem: avMetadataItem) {
+            json["value"] = valueJson
+        }
+        if let existingExtra = avMetadataItem.extraAttributes {
+            var extraAttributes: [String: Any] = [:]
+            for (key, value) in existingExtra {
+                extraAttributes[key.rawValue] = NonFiniteSanitizer.sanitize(value)
+            }
+
+            json["extraAttributes"] = extraAttributes
```

Comment:
nit: you can simplify this with `reduce(into:)`
```suggestion
            var extraAttributes: [String: Any] = [:]
            json["extraAttributes"] = existingExtra.reduce(into: [String: Any]()) { result, entry in
                result[entry.key.rawValue] = NonFiniteSanitizer.sanitize(entry.value)
            }
```